backend/app/exceptions/format.py

```diff
@@ -4,12 +4,8 @@
 class InvalidFormat(Exception):
     """Exception class for invalid file or data format."""
 
-    status_code = 400
-
-    def __init__(self, message, status_code=None, payload=None):
```

Comment:
Not sure where it is used, but you could alternatively just default to status_code=400 in the __init__(self, message, payload=None, status_code=400)